content/en/docs/tasks/traffic-management/ingress/ingress-control/index.md

```diff
@@ -314,3 +314,77 @@ $ kubectl delete gateway httpbin-gateway
 $ kubectl delete virtualservice httpbin
 $ kubectl delete --ignore-not-found=true -f @samples/httpbin/httpbin.yaml@
 {{< /text >}}
+
+## (Experimental) Configuring network toplogies
+
+Istio provides the ability to control X-Forwarded-For and X-Forwarded-Client-Cert as an experimental feature.
+
+Many users choose to deploy Istio ingress gateways in using various network topologies
+(e.g. behind Cloud Load Balancers, a self-managed Load Balancer or directly expose the
+Istio ingress gateway). As such, these topologies require different ingress gateway configurations for
+correct client attributes like IP/ports/certs to be propagated to the workload running in the cluster.
+
+Configuration of XFF and XFCC headers are configured by using by using `MeshConfig` during Istio
+*installation*. To simplify configuration, create a single YAML file to pass to `istioctl`.
+
+{{< text yaml >}}
```

Comment:
@nrjpoddar should we mention Pod annotations?

Reply:
yeah, let's add a section about how gateway workloads can override the default setting.

Additionally, mention that after you have edited the mesh config setting, the gateway pods need to be restarted.

Reply:
Done.